INTEGRATION.md

```diff
@@ -35,6 +35,39 @@ Integration here has 2 different meanings.
 
 2. One way to handle the protocols is to use the [HTTP gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/). This document describes how to determine which gateway to use.
 
+### $CONFIG folder, $GWFILE and $LAGACY_GWFILE
+
+Throughout this spec you might see `$CONFIG/ipfs/...`. `$CONFIG` is a mere abbreviation for the config folder, the variable doesn't exist and only serves to make this spec more readable. This section defines where `$CONFIG` should point to in a cross-platform manner.
+
+As a reference, [this](https://github.com/cjbassi/platform-dirs-rs#path-list) list of configuration folders is used.
+
+| | with variables | full paths |
+| -------- | -------- | -------- |
+| Windows     | %APPDATA%/ipfs     | C:/Users/%USERNAME%/AppData/Roaming/ipfs     |
+| macOS     | $XDG_CONFIG_HOME/ipfs     | ~/Library/Application Support/ipfs     |
+| Linux     | $XDG_CONFIG_HOME/ipfs     | ~/.config/ipfs     |
+
+In the same manner as $CONFIG, $GATEWAY and $LEGACY_GATEWAY are names for this spec only to simplify reading.
+
+`$GWFILE` = `$CONFIG/gateway`
+`$LEGACY_GWFILE` = `~/.ipfs/gateway`
+
+### Gateway file and Legacy gateway file
+#### For KUBO only
```

Comment:
Move this to "Compatibility" section of standalone IPIP in `IPIP` directory.
Kubo's legacy should not pollute the modern specification we want to create here.
We want projects like ffmpeg to follow new spec. Kubo will keep doing `~/.ipfs/gateway` but it should not be something new implementations are even aware of – focus should be on `~/.config/ipfs/gateways` and `/etc/ipfs/gateways` 

Note templates:
- https://github.com/ipfs/specs/blob/main/template.md → living specification
- https://github.com/ipfs/specs/blob/main/IPIP/0000-template.md  → description of introduced changes

(it looks you used IPIP tempalte for living specification – needs clanup, but its mostly moving content between sections – lmk if you need editorial help)